build.js

```diff
@@ -64,12 +65,21 @@ var readLines = function (filename) {
     return togo;
 };
 
+/**
+ * Resolve a templated module file to a file with an absolute path.
+ * @param {String} file - A file path. The path can contain or not contain a template string starting with "%".
+ * @return {String} The actual file location with an absolute path
+ */
+var fileResolver = function (file) {
+    return file.startsWith("%") ? resolve.sync(file.replace(/^%/, "")) : file;
```

Comment:
In the following answer I found an even slightly simpler approach that doesn't require an external dependency - https://stackoverflow.com/a/49455609 - the resolution can simply be require.resolve(file.replace(/^%/, ""));